README.md

```diff
@@ -42,46 +42,176 @@ These elements are described in different folders. All the modelling happens wit
 
 The files that are outside from the `openfisca_country_template` folder are used to set up the development environment.
 
+## Packaging Your Country Package for Distribution
 
-## Installing
+If you wish to distribute your package via `pip install`, follow the steps given by the [Python Packaging Authority](https://python-packaging-user-guide.readthedocs.io/distributing/).
 
-> We recommend that you [use a virtualenv](https://doc.openfisca.fr/for_developers.html#create-a-virtualenv) to install OpenFisca. If you don't, you may need to add `--user` at the end of all commands starting by `pip`.
+If not, edit out the following sections:
```

Comment:
This part seems to imply that one can make a country package without making it a pip compatible distribution.

This is **not** an usage we want to promote. We're not even sure this works, and some tools, like the country package detection in`openfisca-serve`, assume that country packages are always distributions.

Instead of "If you wish", we could have something like: "Country packages are python distributions..."